ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CCDCase.java

```diff
@@ -60,4 +60,5 @@
     private List<CCDCollectionElement<CCDEvidenceRow>> evidence;
     private CCDDocument sealedClaimDocument;
     private String features;
+    private CCDYesNoOption migrated;
```

Comment:
Should we make this name more specific maybe? migratedClaimstorePostgres or migratedV1? We might want to flag future migrations I'm thinking.

Reply:
Renamed. 👍